packages/asc-ui/src/components/Toggle/Toggle.test.tsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render, fireEvent } from '@testing-library/react'
 import Toggle from './Toggle'
 import { KeyboardKeys } from '../../types'
-import ToggleButton from '../Button/ToggleButton/ToggleButton'
 
 describe('Toggle', () => {
-  let component: any
+  let container: any
+  let queryByTestId: any
+  let button: HTMLButtonElement
   const onClickMock = jest.fn()
   beforeEach(() => {
-    component = mount(
+    ;({ container, queryByTestId } = render(
       <Toggle onClick={onClickMock} render={false}>
-        <div id="child">Foo</div>
+        <div data-testid="child">Foo</div>
       </Toggle>,
-    )
+    ))
+    button = container.querySelector('button')
   })
 
   it('should render the toggle button', () => {
-    expect(component.find(ToggleButton).exists()).toBe(true)
+    expect(container.querySelector('button')).toBeDefined()
   })
 
   it('should display the children when the button is clicked', () => {
-    expect(component.find('#child').exists()).toBe(false)
-    component.find(ToggleButton).at(0).simulate('click')
-    expect(component.find('#child').exists()).toBe(true)
+    expect(queryByTestId('child')).toBeNull()
+    fireEvent.click(button)
+    expect(queryByTestId('child')).toBeDefined()
```

Comment:
For testing DOM elements I would use toBeInTheDocument instead or toBeDefined (https://medium.com/better-programming/how-to-do-the-basics-in-react-testing-library-c52cdb3a7726)